Sources/SwiftFormatCore/Trivia+Convenience.swift

```diff
@@ -35,6 +35,20 @@ extension Trivia {
     return count
   }
 
+  /// Returns the number of leading newlines in this trivia.
+  public var numberOfLeadingNewlines: Int {
+    var count = 0
+    for piece in self {
+      if case .newlines(let n) = piece {
```

Comment:
Do we need to generalize this to carriage returns, linefeeds, etc?

Reply:
Done. (This was copied from the property above, which does the same thing, but I don't want to change it yet because it could risk changing something else behaviorally in the formatter. We should audit some of these trivia convenience properties. A few of them are a bit odd, like "get the total number of spaces", which doesn't seem that useful—it's leading or trailing spaces that you usually want.)